bedrock/firefox/templates/firefox/whatsnew/base-pebbles.html

```diff
@@ -0,0 +1,32 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{% block gtm_page_id %}data-gtm-page-id="/firefox/whatsnew/"{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}{{ _('A better, faster, private browser for today') }} | {{ _('Firefox') }}{% endblock %}
```

Comment:
I took these strings from the `/firefox` page, as the meta sharing strings on older whatsnew pages are sorely out of date and no one ever gives us new ones. This template is what will hopefully be a new base template for all future `/whatsnew` pages, so it might be worth putting these strings into a shared lang file?

Reply:
My understanding is that we're targeting only German for 56. Isn't that the case (@peiying2)?

Reply:
OK, after trying to extract the strings I have a better picture. This has nothing to do with 56 being (or not) only for German.

Before exposing them to a shared file, I think it would be good to get some confirmation that it's useful. In other words, I don't want to add these strings to discover that no template will ever use them in the future.

If we start getting whatsnew pages using them, we can move them to main.lang

Reply:
> Before exposing them to a shared file, I think it would be good to get some confirmation that it's useful. In other words, I don't want to add these strings to discover that no template will ever use them in the future.

Well, we need to start transitioning `/whatsnew` pages to use Pebbles instead of Sandstone, and this new base template enables that so future pages can build upon it. The meta strings included here are just for social sharing, but having them here means we no longer need to copy/paste them into each new template. I can't promise they won't ever change in the future, but I can at least run these strings past our copy writer today. 

> If we start getting whatsnew pages using them, we can move them to main.lang

Makes sense

Reply:
I missed these strings were for OG sharing, not the title/desc of the page. Adding them to main.lang, to avoid referencing the lang file in base.